src/_css/Card.scss

```diff
@@ -20,6 +20,7 @@
   padding: 20px;
   margin: 0 20px 20px 0;
   border: 0;
+  outline: none;
```

Comment:
Removing outline is not recommended for accessibility purpose

Reply:
Okay. I'll revert it. Though it looks very weird with the outline